jdtls.ext/com.shengchen.checkstyle.runner/src/main/java/com/shengchen/checkstyle/quickfix/whitepace/MethodParamPadQuickFix.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) jdneo
+
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.shengchen.checkstyle.quickfix.whitepace;
+
+import com.shengchen.checkstyle.quickfix.BaseEditQuickFix;
+
+import org.eclipse.jface.text.BadLocationException;
+import org.eclipse.jface.text.Document;
+import org.eclipse.jface.text.IRegion;
+import org.eclipse.text.edits.DeleteEdit;
+import org.eclipse.text.edits.InsertEdit;
+import org.eclipse.text.edits.TextEdit;
+
+/**
+ * Quick fix for
+ * https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/checks/whitespace/MethodParamPadCheck.html
+ */
+public class MethodParamPadQuickFix extends BaseEditQuickFix {
```

Comment:
Seems not handle well with:

```xml
<module name="MethodParamPad">
  <property name="tokens" value="METHOD_DEF"/>
  <property name="option" value="space"/>
  <property name="allowLineBreaks" value="true"/>
</module>
```

![image](https://user-images.githubusercontent.com/6193897/106238704-4b8ca380-623c-11eb-9f86-83eca6205ceb.png)

Reply:
@jdneo when I test this, Checkstyle reports that the "(" is not preceded by whitespace, and the quick fix adds a space before the "(". That seems like it's doing the right thing to me?